docs/core/install/linux-ubuntu.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Install .NET Core on Ubuntu - .NET Core
+description: Demonstrates the various ways to install .NET Core SDK and .NET Core Runtime on Ubuntu.
+author: thraka
+ms.author: adegeo
+ms.date: 05/18/2020
+---
+
+# Install .NET Core SDK and .NET Core Runtime on Ubuntu
+
+.NET Core is supported on different Ubuntu versions. This article describes how to install .NET Core on Ubuntu with APT. When an Ubuntu version falls out of support, .NET Core is no longer supported with that version. However, these instructions may help you to get .NET Core running on those versions, even though it isn't supported.
+
+## Supported distributions
+
+The following is a list of currently supported .NET Core releases and the versions of Ubuntu they're supported on. These versions remain supported until either the version of [.NET Core reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of [Ubuntu reaches end-of-life](https://wiki.ubuntu.com/Releases).
+
+For the best compatibility, choose a long-term release (LTS) version.
+
+| .NET Core | Ubuntu version (LTS) | Ubuntu version |
+|-----------|----------------------|----------------|
+| 2.1 (LTS) | 16.04, 18.04, 20.04  | 19.10          |
+| 3.1 (LTS) | 16.04, 18.04, 20.04  | 19.10          |
+
+.NET Core versions no longer supported and the Ubuntu LTS releases they were supported with:
+
+| .NET Core | Ubuntu version (LTS) |
+|-----------|----------------------|
+| 3.0       | 16.04, 18.04         |
```

Comment:
Why doesn't this include non-LTS Ubuntu versions?

Reply:
I didn't think a non-supported ubuntu version coupled with a non-supported .NET Core would be as relevant. I can get that information though and add it if you think it's useful.

Reply:
I like the idea of only including Ubuntu LTS releases in the 'previous .NET versions' table.

Reply:
I'm a little averse to making it hard to find this info, but I agree, it's not as relevant.